host/xtest/xtest_1000.c

```diff
@@ -1130,8 +1127,6 @@ static void xtest_tee_test_1013(ADBG_Case_t *c)
 			max_concurrency = arg[n].max_concurrency;
 	}
 
-	Do_ADBG_Log("Max concurrency %zu", max_concurrency);
```

Comment:
This information is interesting to see, perhaps print it after the loop calling this function is ended. I guess we're interested in the average max concurrency.